Source/Core/Core/Core.cpp

```diff
@@ -62,6 +62,12 @@
 #include "VideoCommon/OnScreenDisplay.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+#ifdef _MSC_VER
```

Comment:
gcc 4.8 (which is our current minimum version) apparently supports thread_local. Looks like VS2015 supports it too. 

Might want to just define thread_local on vs2013 and leave it for all other compilers.